infrastructure/kubernetes-demo/template.yaml

```diff
@@ -242,3 +242,46 @@ Resources:
           - elasticloadbalancing:RemoveListenerCertificates
           - elasticloadbalancing:ModifyRule
           Resource: "*"
+
+  HelloWorldDocumentDBCluster:
+    Type: AWS::DocDB::DBCluster
+    DeletionPolicy: Delete
```

Comment:
Interesting, what's a DeletionPolicy?

Reply:
Gotcha: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-deletionpolicy.html

Right so this seems like a parameter we can use in a bunch of places... So for a database we may want a policy where I can delete the service, but retain or back up the data. Nice.

Reply:
Yes, or set it to `Retain` so if the stack is removed the database stays up in a form of deletion protection. I figured we aren't going to care about this demo data though.